scripts/test/Muon/elemental_analysis/periodic_combo_test.py

```diff
@@ -0,0 +1,77 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import print_function, absolute_import
+
+import unittest
+
+from mantid.py3compat import mock
+from mantidqt.utils.qt.testing import start_qapplication
+
+import Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table as periodic_table
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table import PeriodicTableItem, PeriodicCombo
+
+
+@start_qapplication
+class PeriodicComboTest(unittest.TestCase):
+    @mock.patch('Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table.QtWidgets.QComboBox.insertItem')
+    def test_that_init_defaults_to_all_elements_in_periodic_table(self, mock_insert_item):
+        PeriodicCombo()
+        self.assertEqual(mock_insert_item.call_count, 109)
+        mock_insert_item.assert_called_with(108, 'Mt (109) - meitnerium')
+
+        PeriodicCombo(detailed=False)
+        self.assertEqual(mock_insert_item.call_count, 109+109)
```

Comment:
are we really call this 218 times? That does not seem right

Reply:
The constructor in `PeriodicCombo` loops over all elements of the periodic table, 109 in total. In the test this is called twice, so `insertItem` is called 218 times.
I split up the test in two cases of 109 each.